### PR TITLE
Labels support

### PR DESCRIPTION
**Pull request by fredrikmikkelsen**
- Added support for changing Select's labels

### CODE REVIEW

File: src/DynamicSelect.php [hunk 1]
```diff
@@ -27,6 +31,10 @@ public function meta()
             'options' => $this->getOptions($this->dependentValues),
             'dependsOn' => $this->getDependsOn(),
             'dependValues' => $this->dependentValues,
+            'placeholder' => $this->placeholder,
```
Comment (royduin) [inline src/DynamicSelect.php]: Why not?
```
'placeholder' => __('Pick a value'),
```

Comment (fredrikmikkelsen) [reply src/DynamicSelect.php]: My thought was that it gives more freedom to customize the label - for example if a user wanted the label value to be "Choose an option" as in the default Select field.

Comment (royduin) [reply src/DynamicSelect.php]: But now we don't have any default translations, I think you can use `withMeta()` to overwrite the translations:

```
DynamicSelect::make('Country', 'country')
    ->options(['US' => 'United States', 'UK' => 'United Kingdom'])
    ->withMeta([
        'placeholder' => 'Choose an option'
    ])
```

Comment (fredrikmikkelsen) [reply src/DynamicSelect.php]: Makes sense. I've changed it to use localization.

But
`->withMeta([
        'placeholder' => 'Choose an option'
    ])`
doesn't work.

Comment (royduin) [reply src/DynamicSelect.php]: It works perfectly here...

**Comment by royduin** [comment]
Released in 1.3.0 🚀

File: src/DynamicSelect.php (hunk 1 above)
Comment (fredrikmikkelsen) [reply src/DynamicSelect.php]: It now works here as well. Thanks again.

Comment (royduin) [reply src/DynamicSelect.php]: Nice, good to hear!

**Pull request merged**